pkg/util/domainutil/repair_vars.go

```diff
@@ -101,7 +103,7 @@ func (r *repairInfo) GetRepairedTableInfoByTableName(schemaLowerName, tableLower
 		if db.Name.L != schemaLowerName {
 			continue
 		}
-		for _, t := range db.Tables {
+		for _, t := range db.Tables() {
```

Comment:
I see a preprocesser function `handleRepairName` still calls `GetRepairedTableInfoByTableName` but it can't ensure it's v1 or v2

Reply:
Here it's a little bit special, repare table still need to get the Tables field.